src/openrct2/world/Map.cpp

```diff
@@ -2627,8 +2393,8 @@ void game_command_place_large_scenery(
 
         if (!gCheatsDisableClearanceChecks
             && !map_can_construct_with_clear_at(
-                   curTile.x, curTile.y, zLow, zHigh, &map_place_scenery_clear_func, bl, flags, &supportsCost,
-                   CREATE_CROSSING_MODE_NONE))
+                curTile.x, curTile.y, zLow, zHigh, &map_place_scenery_clear_func, bl, flags, &supportsCost,
```

Comment:
Looks like you have an autoformatter on that has different settings than clang-format. It's better to turn it off and just do `git clang-format` after making changes and doing `git add`.

Reply:
I'm using clang format just it doesn't like to match the other clang format.